Prefix key to fetch html with namespace

kategengler opened the pull request:
- The value of the ":current" key put there by ember-cli-deploy is the
  revision, not the redis key. See https://github.com/ember-cli-deploy/ember-cli-deploy-redis/blob/master/tests/unit/lib/redis-nodetest.js#L191

Fixes #2

kategengler committed:
Prefix key to fetch html with namespace
- The value of the ":current" key put there by ember-cli-deploy is the
  revision, not the redis key

seanpdoyle commented:
Hey @kategengler, thanks for opening this issue and bringing this to our attention.

Do you use this gem on its own, or are you also using [`ember-cli-rails`](https://github.com/thoughtbot/ember-cli-rails)?

I ask because we recently opened a PR (https://github.com/seanpdoyle/ember-cli-rails-deploy-redis/pull/4) to integrate more deeply with `ember-cli-rails`.

That PR incorporates these changes, but comes with lots of breaking changes. To continue to use this gem in its current form, you'd have to reimplement the [Page](https://github.com/seanpdoyle/ember-cli-rails-deploy-redis/blob/d5b8acabacb6864a8b7fea5af00533eae7ef81a3/lib/ember-cli/deploy/page.rb) class.

I'll merge this PR as part of an `0.0.2` release, but it is likely there will be breaking changes coming in an `0.1.0` version. I just wanted to give you a heads up and get some user feedback.

If you haven't used `ember-cli-rails` in the past (or if you have and had a negative experience), we've just released `0.6.0` which resolves all the issues we've had in the past.

Please try it out: https://github.com/thoughtbot/ember-cli-rails

seanpdoyle commented:
Released in https://github.com/seanpdoyle/ember-cli-rails-deploy-redis/releases/tag/v0.0.2

kategengler commented:
Hi @seanpdoyle - The project I used this gem for had actually been on `ember-cli-rails` and moved off of it because they wanted the application in a separate repo and to deploy it on different schedule from the Rails app. This gem nicely worked great as a counterpart to `ember-cli-deploy-lightning-pack` for handling everything Rails side and making use of `append_to_head`. On previous projects, I implemented those features myself and could easily go back to doing that or maintain a fork If this gem is going to be intended to be used with `ember-cli-rails` only.

seanpdoyle commented:
@kategengler I've extracted the [`html_page`](https://github.com/seanpdoyle/html_page) gem from [`ember-cli-rails`](https://github.com/thoughtbot/ember-cli-rails/pull/356).

I've also added a section to the https://github.com/seanpdoyle/ember-cli-rails-deploy-redis/pull/4 `README.md` explaining how to achieve the behavior you're describing **without** `ember-cli-rails`.

Would you mind providing feedback on https://github.com/seanpdoyle/ember-cli-rails-deploy-redis/pull/4?
